cluster/examples/aws-setup-provider/setup.sh

```diff
@@ -36,7 +36,7 @@ kubectl cluster-info > /dev/null || echo "KUBECONFIG is not configured properly"
 aws_profile="${aws_profile:-default}"
 
 # if region is not provided, retrieve aws profile region from config
-AWS_REGION=$(awk '/["$aws_profile"]/ {getline; print $3}' ${HOME}/.aws/config)
+AWS_REGION=$(cat ${HOME}/.aws/config | awk '/["$aws_profile"]/ {getline; print $0}' | awk '$1=="region"{print $3}')
```

Comment:
I just realized we use `aws configure get region --profile ${aws_profile}` for this in docs. I think that could be better. See https://crossplane.io/docs/v0.8/cloud-providers/aws/aws-provider.html

Reply:
@muvaf I've updated the code with your suggestion (which was brilliant). I did not understand what you meant by using it in the docs. Did you mean code?

Reply:
Oh I meant we're already using that command in the docs. For some reason, it's different how we populate `AWS_REGION` here in this script and in the docs that let you do these manually.